lib/internal/v8/startup_snapshot.js

```diff
@@ -54,8 +54,6 @@ function runSerializeCallbacks() {
     const { 0: callback, 1: data } = serializeCallbacks.shift();
     callback(data);
   }
-  // Remove the hooks from the snapshot.
-  require('v8').startupSnapshot = undefined;
```

Comment:
Why is this line removed?

Reply:
We need this to always present for `isErrorStackTraceLimitWritable()` - could be replaced by internal namespaces, but then this removal is simply not used anywhere and undocumented - probably something that I left behind when I landed the original PR. Checking `require('v8').startupSnapshot.isBuildingSnapshot()` before using any of the APIs should already be enough for users anyway - other methods from the API simply throw an error if that's false.

Reply:
Sounds good to me. Probably a test on the return value `require('v8').startupSnapshot.isBuildingSnapshot()` after deserialization is worth to cover.